### PR TITLE
Blog webpack performance

### PR DESCRIPTION
**Pull request by adamkozuch**

### CODE REVIEW

File: blog/posts/2017-05-30.markdown [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+unused exports in it. Also, I will briefly describe code splitting feature and
+how brand new `performance` configuration object can help you to use this feature properly.
+
+## Tree Shaking
+I would like to start from tree shaking feature which during transpiling process exclude
+unused exports. This very simple example will illustrate the issue: 
+
+```
+// exports.js
+exports function foo() {
```
Comment (asial-matagawa) [inline blog/posts/2017-05-30.markdown]: I think `exports function` should be `export function`.
In ES2015, we use `export` instead of `exports`.

Comment (asial-matagawa) [inline blog/posts/2017-05-30.markdown]: @khemry @adamkozuch 
Sorry for the delay.
Could you fix this part in the blog post?

Comment (khemry) [inline blog/posts/2017-05-30.markdown]: @asial-matagawa Thanks for your feedback. I think @adamkozuch will fix this tomorrow.

Comment (asial-matagawa) [inline blog/posts/2017-05-30.markdown]: Thank you!

Comment (adamkozuch) [inline blog/posts/2017-05-30.markdown]: @asial-matagawa thank you for feedback. Fixed.

Comment (asial-matagawa) [inline blog/posts/2017-05-30.markdown]: 👍

**Pull request merged**